CHANGELOG.md

```diff
@@ -21,8 +21,8 @@
 - Migrate `ReportsScreen` to a fragment
 - Migrate `OverdueScreen` to a fragment
 - Add `DatePickerKeyFactory` for getting date picker key based on `Country`
+- Migrate app to use ViewBinding
```

Comment:
This PR also includes removal of the `kotlin-android-extensions` and replacement with `kotlin-parcelize`. Can we include this change as well?

Reply:
This also needs to be in the "Next Release" section.

Reply:
>This also needs to be in the "Next Release" section.

I got confused because the in-progress items were present in the `On Demo` section. Didn't see the title 😅